driver/src/models/flux.rs

```diff
@@ -0,0 +1,90 @@
+use byteorder::{BigEndian, WriteBytesExt};
+use serde_derive::{Deserialize, Serialize};
+use web3::types::H256;
+
+use crate::models::{Serializable, RootHashable, merkleize};
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Debug, Ord, PartialOrd, Eq)]
+#[serde(rename_all = "camelCase")]
```

Comment:
I still don't really understand what this is...?

Reply:
When serializing it to json it turns the snake_case field names into camelCase.

E.g. the struct
```
{field_a: "foo"}
```

would become the json

```
{"fieldA": "foo"}
```

This is needed, because python uses camel case and rust uses snake case by convention for variable names. Thus python writes camelCased json objects into our DB. In order to use snake_cased variable names in rust while still automatically converting what we find in the database, this transformation is needed.

Reply:
> python uses camel case and rust uses snake case by convention for variable names

In fact, both use snake case for variables (and functions)

Sweet I get it now. thanks.

Reply:
Interesting, than just the conversion into json is done with camelCase in python. We could thus also change all of this code (https://github.com/gnosis/dex-services/blob/master/event_listener/dfusion_db/models.py#L51-L55) to use snake case in the db.